crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -128,6 +132,7 @@ pub struct LiquidityRequest {
 }
 
 /// An LN-DLC node.
+#[derive()]
```

Comment:
❓ Is this doing anything or can that be removed?

Reply:
Nice catch!